config/nvim/lua/plugins/lang.lua

```diff
@@ -33,4 +33,26 @@ return {
       },
     },
   },
+
+  -- Lua
+  {
+    "rafcamlet/nvim-luapad",
+    cmd = { "Luapad" },
+    ft = "lua",
+    opts = { eval_on_change = false },
+    keys = {
+      { "<localleader>p", "<cmd>Luapad<cr>", mode = "n" },
+      { "<localleader>r", "<cmd>LuaRun<cr>", mode = "n" },
+    },
```

Comment:
remove this 
```suggestion
```

Reply:
fiewuf hiwefuhi weufiwefuhiweu fhiuwefi
# hello 
ifuhweif uweifhweuifhweuifhwefu

## fhweiufhiwefuhwief 

fweuhfiwuefhiwefuhweifuwe

Reply:
fiewuf hiwefuhi weufiwefuhiweu fhiuwefi
# hello 
ifuhweif uweifhweuifhweuifhwefu

## fhweiufhiwefuhwief 

```suggestion
    keys = {
      { "<leader>p", "<cmd>Luapad<cr>", mode = "n" },
      { "<localleader>r", "<cmd>LuaRun<cr>", mode = "n" },
    },
```